modules/security/src/main/java/org/eclipse/xpanse/modules/security/config/WebSecurityConfig.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.security.config;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.web.SecurityFilterChain;
+
+/**
+ * Configuration applied on all web endpoints defined for this
+ * application. Any configuration on specific resources is applied
+ * in addition to these global rules.
+ */
+@Slf4j
+@Profile("zitadel")
+@Configuration
+@EnableWebSecurity
+public class WebSecurityConfig {
```

Comment:
Since it is only zitadel specific config, can we add 'Zitadel' to the classname?

Reply:
OK